modules/apps/portal-vulcan/portal-vulcan-api/src/main/java/com/liferay/portal/vulcan/problem/Problem.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.portal.vulcan.problem;
+
+import java.util.Locale;
+
+/**
+ * @author Alejandro Tardín
+ */
+public interface Problem {
+
+	public String getDetail(Locale locale);
+
+	public Status getStatus(Locale locale);
```

Comment:
Does the status depends on the locale? :thinking:

Reply:
There is a reason phrase that could be localized.

Reply:
Yes, but the status?

Reply:
As discussed we will remove it.